src/cmds/route.rs

```diff
@@ -17,15 +19,68 @@ pub async fn list_routes(args: ListRoutes) -> Result<Msg> {
 
 pub async fn get_route(args: GetRoute) -> Result<Msg> {
     let mut client = client::RouteClient::new(&args.config_host, &args.config_pubkey).await?;
+    let stats_str = if args.stats {
+        let RouteStats {
+            devaddr_count,
+            eui_count,
+            skf_count,
+        } = counts_for_route(
+            &args.config_host,
+            &args.config_pubkey,
+            &args.route_id,
+            &args.keypair.to_keypair()?,
+        )
+        .await?;
+        format!(
+            "\nDevAddr Range: {}\nEUI Count: {}\nSKF Count: {}",
```

Comment:
probably better to save for a later change and make the shift all in one big go but formatting these outputs as json would probably make for a more versatile tool. we can have the stats struct derive serde_json implementations and then use the `json!` macro to format the outputs as json for pretty-printing that let the caller quickly parse the result with a tool like jq if they're too lazy to write a tool that actually calls the grpc apis and they just want to script calling the config-cli and piping into something

Reply:
agreed. I might make sense to have a stats rpc. That would also be useful in hpr land for not needing to stream an entire routes details just to check if there's a significant difference.